src/content/docs/new-relic-solutions/tutorials/add-visualization-dashboard.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: "Add your custom visualization to a dashboard"
+metaDescription: "Add your custom visualization to a New Relic dashboard"
+freshnessValidatedDate: never
+---
+
+
+
+<Callout variant="tip">
+
+This lesson is part of a course that teaches you how to build a custom visualization on the New Relic platform.
+
+Each lesson in the course builds upon the last, so make sure you've completed the last lesson, [Customize visualizations with configuration](/docs/new-relic-solutions/tutorials/custom-visualization-config), before starting this one.
+
+</Callout>
+
+
+In the previous lesson, you added a configuration option to your visualization that allows you to choose how you want to represent your data. This was different from the previous version of your code because, while you can still choose your chart type, you can now only do it before you create an instance of your visualization. But what does it mean to create an instance of your visualization? How do you use a visualization once you've configured it?
+
+In this lesson, you'll learn how to create an instance of your visualization and add that instance to dashboard.
+
+<Callout variant="tip">
+
+If you got lost in the previous steps and would like to see what your files should look like to start this lesson, check out the [course project](https://github.com/newrelic/nru-custom-visualization-course) on Github.
+
+</Callout>
+
+
+## Add your custom visualization to a dashboard [#add-dashboard]
+
+<Steps>
+
+<Step>
+
+From the root directory of your Nerdpack, publish and subscribe to your visualization:
+
+```sh
+nr1 nerdpack:publish
+nr1 nerdpack:subscribe
+```
+
+<Callout variant="important">
+
+To publish a Nerdpack you must be a Full user with the Nerdpacks “modify” privilege (this is granted by default).
+
+Read our [Nerdpacks privileges](/build-apps/permission-manage-apps/) documentation to learn more.
```

Comment:
This is a broken link

Reply:
This is the messiness of migrating hundreds of docs in one go.

```suggestion
```